AnagramSolver.Cli/Program.cs

```diff
@@ -9,45 +9,46 @@ class Program
     {
         static void Main(string[] args)
         {
-            var builder = new ConfigurationBuilder()
-               .AddJsonFile($"appsettings.json", true, true);
-            var config = builder.Build();
-            var minLength = config["MinWordLength"];
-            int minLengthAsNumber = Int32.Parse(minLength);
-            var maxAnagrams = config["MaxNumberOfAnagrams"];
-            int maxAnagramsAsNumber = Int32.Parse(maxAnagrams);
-            Console.WriteLine(maxAnagramsAsNumber);
-
+            int minLength, maxAnagrams;
+            string filePath;
+            ConfigureAppSettings(out minLength, out maxAnagrams, out filePath);
```

Comment:
Out is not a bad practice when used correctly. Here we can see that if we apply more configuration, we have to expand with new parameter and make a variable for it. There are other ways of structuring multiple fields together  (wink wink) and using methods return value.

Reply:
okay